src/quo2/components/community/banner/style.cljs

```diff
@@ -2,18 +2,21 @@
   (:require [quo2.foundations.colors :as colors]))
 
 (defn community-card
-  [radius]
-  {:shadow-offset    {:width  0
-                      :height 2}
-   :shadow-radius    radius
-   :shadow-opacity   1
-   :shadow-color     colors/shadow
-   :elevation        1
-   :border-radius    radius
-   :justify-content  :space-between
-   :background-color (colors/theme-colors
-                      colors/white
-                      colors/neutral-90)})
+  []
+  {:shadow-offset     {:width 0 :height 2}
+   :shadow-radius     16
+   :shadow-opacity    1
+   :shadow-color      colors/shadow
+   :elevation         1
+   :border-radius     16
+   :justify-content   :space-between
+   :background-color  (colors/theme-colors colors/white colors/neutral-90)
+   :flex-direction    :row
+   :margin-horizontal 20
+   :margin-top        8
+   :margin-bottom     8
```

Comment:
Can be `:margin-vertical 8`

Reply:
If we set `margin-vertical 8` when we override the styles of a single margin (e.g., `margin-top 10`), it will be ignored since RN prefers `margin-vertical`.

For this PR, I'm overriding a single margin.

Reply:
Thanks for the explanation Ulises!
Ok